### PR TITLE
Fix password

### PR DESCRIPTION
**Pull request by NyakudyaA**
* Generate random password for the role superuser and replication when env variable is not set
* Password is visible in the logs and as a text file within the container
* Adjusted scenario test to explicitly require the password
* Added kartoza figlet to the image to advertise the image
* Added env POSTGRES_INITDB_WALDIR which allows wal files to be stored outside data dir as per https://github.com/docker-library/postgres/blob/master/13/alpine/docker-entrypoint.sh#L48

### CODE REVIEW

File: scripts/env-data.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@ POSTGRES_MAJOR_VERSION=$(cat /tmp/pg_version.txt)
 POSTGIS_MAJOR=$(cat /tmp/pg_major_version.txt)
 POSTGIS_MINOR_RELEASE=$(cat /tmp/pg_minor_version.txt)
 DEFAULT_DATADIR="/var/lib/postgresql/${POSTGRES_MAJOR_VERSION}/main"
+POSTGRES_INITDB_WALDIR="/opt/postgresql/${POSTGRES_MAJOR_VERSION}/pg_waldir"
```
Comment (lucernae) [inline scripts/env-data.sh:6]: So, to reconfirm, `POSTGRES_INITDB_WALDIR` uses `/opt/postgresql` directory while `DEFAULT_DATADIR` uses `/var/lib/postgresql` ?

**Review by lucernae — COMMENTED** [review]
LGTM, it seems there only slight refactoring changes.

File: scripts/env-data.sh (hunk 1 above)
Comment (NyakudyaA) [reply scripts/env-data.sh:6]: Yes, I can change the location from /opt/postgresql to another location if you feel so. The idea is to be able to mount the two in different folders as done in  Postgres official images

File: README.md [hunk 2]
```diff
@@ -306,6 +306,10 @@ To create a running container do:
 docker run --name "postgis" -p 25432:5432 -d -t kartoza/postgis
 ```
 
+**NB** If you do not pass the env variable `POSTGRES_PASS` a random password
```
Comment (timlinux) [inline README.md]: Replace NB with Note (many people are unfamiliar with the term notae benae)

**Review by timlinux — APPROVED** [review]

**Pull request merged**